composer.json

```diff
@@ -26,44 +26,45 @@
         "source": "https://github.com/nuwave/lighthouse"
     },
     "require": {
-        "php": ">= 7.2",
+        "php": ">= 7.4",
         "ext-json": "*",
         "haydenpierce/class-finder": "^0.4",
-        "illuminate/auth": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "illuminate/bus": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "illuminate/contracts": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "illuminate/http": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "illuminate/pagination": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "illuminate/queue": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "illuminate/routing": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "illuminate/support": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "illuminate/validation": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "laragraph/utils": "^1",
+        "illuminate/auth": "^8.13 || ^9",
+        "illuminate/bus": "^8.13 || ^9",
+        "illuminate/contracts": "^8.13 || ^9",
+        "illuminate/http": "^8.13 || ^9",
+        "illuminate/pagination": "^8.13 || ^9",
+        "illuminate/queue": "^8.13 || ^9",
+        "illuminate/routing": "^8.13 || ^9",
+        "illuminate/support": "^8.13 || ^9",
+        "illuminate/validation": "^8.13 || ^9",
+        "laragraph/utils": "dev-graphql-php-15",
         "thecodingmachine/safe": "^1 || ^2",
-        "webonyx/graphql-php": "^14.7"
+        "webonyx/graphql-php": "^15"
     },
     "require-dev": {
         "algolia/algoliasearch-client-php": "^3 || ^4",
-        "bensampo/laravel-enum": "^1.28.3 || ^2 || ^3 || ^4 || ^5 || ^6",
+        "bensampo/laravel-enum": "^3 || ^4 || ^5 || ^6",
         "ergebnis/composer-normalize": "^2.2.2",
-        "laravel/framework": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9",
-        "laravel/legacy-factories": "^1",
-        "laravel/lumen-framework": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8 || ^9 || dev-master",
-        "laravel/scout": "^7 || ^8 || ^9",
-        "mll-lab/graphql-php-scalars": "^4 || ^5",
+        "laravel/framework": "^8.13 || ^9",
+        "laravel/legacy-factories": "^1.1.1",
+        "laravel/lumen-framework": "^8 || ^9 || dev-master",
+        "laravel/scout": "^8 || ^9",
+        "mll-lab/graphql-php-scalars": "dev-graphql-php-15",
         "mll-lab/php-cs-fixer-config": "^4.4.1",
-        "mockery/mockery": "^1",
-        "nunomaduro/larastan": "^0.6 || ^0.7 || ^1 || ^2",
-        "orchestra/testbench": "3.6.* || 3.7.* || 3.8.* || 3.9.* || ^4 || ^5 || ^6 || ^7",
-        "phpbench/phpbench": "^1",
-        "phpstan/phpstan": "~0.12 || ^1",
-        "phpstan/phpstan-mockery": "~0.12 || ^1",
-        "phpstan/phpstan-phpunit": "~0.12 || ^1",
-        "phpunit/phpunit": "^7.5 || ^8.4 || ^9",
+        "mockery/mockery": "^1.5",
+        "nesbot/carbon": "^2.62.1",
+        "nunomaduro/larastan": "^0.7 || ^1 || ^2",
```

Comment:
As php 8 is now required, I think we can tighten this dev dependency to a newer version

Reply:
We only require PHP 7.4, but a bump of larastan/phpstan was possible nonetheless.